src/tv2_afvd_showstyle/getRundown.ts

```diff
@@ -131,29 +131,24 @@ function getGlobalAdLibPiecesAFKD(context: IStudioUserContext, config: Blueprint
 							}
 						})
 					}),
-					literal<TSR.TimelineObjSisyfosChannels & TimelineBlueprintExt>({
-						id: '',
-						enable: {
-							start: 0
-						},
-						priority: 1,
-						layer: SisyfosLLAyer.SisyfosPersistedLevels,
-						content: {
-							deviceType: TSR.DeviceType.SISYFOS,
-							type: TSR.TimelineContentTypeSisyfos.CHANNELS,
-							overridePriority: 1,
-							channels: config.stickyLayers
-								.filter(layer => !info.sisyfosLayers || !info.sisyfosLayers.includes(layer))
```

Comment:
I think this filtering should be preserved, otherwise we might accidentally silence some channels, if some of the sticky layer channels were also chosen in the Sisyfos layers dropdown for an evs.
![Screenshot 2022-01-26 at 11 27 13](https://user-images.githubusercontent.com/9103996/151151960-4a855005-495c-4eef-8bd0-29ac9b867cb0.png)

Reply:
Agree. 
I did make the change, but forgot to commit it. 
It is now commited